### DISCUSSION TITLE
How can I remove the installments input?

### DISCUSSION THREAD

**Discussion by Pacocc97** [mercadopago/sdk-js#167] [Q&A] [answered]
I´m following the bricks checkout guide, I didn´t set a maxInstallment value (as shown in example) but is still showing in my page, and also the min payment option is 3 months!!
heeeeelp.
![Captura de pantalla 2023-06-15 124727](https://github.com/mercadopago/sdk-js/assets/108708176/7c8ac3e2-a449-471e-8a35-f789a090cb9d)

**Comment by Pacocc97** [comment]
And when I add the maxInstallments to the customization, I get an error of incomplete form!
`customization: {
      paymentMethods: {
        minInstallments: 1,
        maxInstallments: 1,
      },
      visual: {
        style: {
          theme: "default", // | 'dark' | 'bootstrap' | 'flat'
        },
      },
    },`
![image](https://github.com/mercadopago/sdk-js/assets/108708176/a531aa44-2e42-4c74-a425-530e552b262c)

**Comment by icaldana** [comment, marked as answer]
Hey @Pacocc97, thanks for using Bricks!

It seems there are two things happening here

1. If you configure `minInstallments` and `maxInstallments` to 1, the installment selection won’t show, since there is only one option. For more details please check [this link](https://www.mercadopago.com.br/developers/en/docs/checkout-bricks/payment-brick/additional-customization/configure-installments).
2. The second point is, please make sure you are testing a transaction with a significant amount, there are minimal amounts required for each country, if you are testing the integration, try to increase the transaction value to above the minimal. 

I hope that helps!

**Reply by Pacocc97** [reply]
Yes! I did "If you configure minInstallments and maxInstallments to 1", and with that it doesnt render that field, but when I submit the card form, (with the minInstallments and maxInstallments ) it toast a message saying the I need to fill all fields to continue. My guess is that it is not detecting the installments value, even though I´ve already setted to 1.

**Reply by Pacocc97** [reply]
Never mind, you where right. It was the total amount! thanks haha

**Comment by Pacocc97** [comment]
const renderCardPaymentBrick = async (bricksBuilder) => {
  const settings = {
    initialization: {
      amount: cartTotal, // monto a ser pago
      payer: {
        email: "",
      },
    },
    customization: {
      paymentMethods: {
        minInstallments: 1,
        maxInstallments: 1,
      },
      visual: {
        style: {
          theme: "default", // | 'dark' | 'bootstrap' | 'flat'
        },
      },
    },
    callbacks: {
      onReady: () => {
        // callback llamado cuando Brick esté listo
      },
      onSubmit: (cardFormData) => {
        //  callback llamado cuando el usuario haga clic en el botón enviar los datos
        //  ejemplo de envío de los datos recolectados por el Brick a su servidor
        console.log(cardFormData, "cardFormData");
        return new Promise((resolve, reject) => {
          fetch("https://api.mercadopago.com/v1/payments", {
            method: "POST",
            headers: {
              "Content-Type": "application/json",
              Authorization: `Bearer TEST-6233838149781149-101811-c8c640262c6d006a398496c170cb4760-21650101`,
            },
            body: JSON.stringify(cardFormData),
          })
            .then((response) => response.json())
            .then((response) => {
              console.log(response, "response");
              // recibir el resultado del pago
              resolve();
            })
            .catch((error) => {
              console.log(error);

              // tratar respuesta de error al intentar crear el pago
              reject();
            });
        });
      },
      onError: (error) => {
        console.log(error);
        // callback llamado para todos los casos de error de Brick
      },
    },
  };
  window.cardPaymentBrickController = await bricksBuilder.create(
    "cardPayment",
    "cardPaymentBrick_container",
    settings
  );
};
